### DISCUSSION TITLE
Plug-in not available in version 6.0.0.20753

### DISCUSSION THREAD

**Discussion by SeatLeonn** [calibx/msfsdeck#253] [Q&A]
Hello,

I just updated to version 6.0.0.20753, but now I can't find the plug-in in the marketplace. Am I doing something wrong? I have been using this plug-in in previous versions.

**Comment by lxndrb22** [comment]
Hello,
I ran into the exact same problem and it seems that things are broken with the new release. 
It would be highly apreciated if it gets updated to the new version, because it was really very well done.
So if you are hearing us Calibx...

**Comment by calibx** [comment]
Hello,

I can confirm the problem : https://github.com/calibx/msfsdeck/issues/252
I hope to find a fix before MSFS 2024 :p

**Reply by SeatLeonn** [reply]
Thanks for your response and efforts to resolve this. 

Will this plug-in still work for msfs2020 if 2024 is out? 

Kind regards

**Reply by lxndrb22** [reply]
I am so glad to hear that as well and it is impressive how quickly you responded.

Kind regards

**Comment by NJTravelerDH** [comment]
I think I have the same problem - MSFS 2020, no values appear in plugin and dials do not work.
I hope this is the right place for this discussion.  If not, we / I can move it.
Loupedeck Live 6.0..., did not work, Downgraded to 5.9.1.
Latest Calibx plugin, upgraded to 3.0.1, required MobiFlight, so I added MobiFlight 10.3.0 (which was upgraded to 10.4.0).
My system Win 11, Honeycomb Throttle Quadrant, Thrustmster T.16000M, & CH Pro Pedals plus above software.
Help requests to SimHanger, MobiFlight, & Loupedeck all suggest help from plugin author &/or supporters.
MobiFlight Module error in lower left corner states no modules found.
MobiFlight Module Tab at top states MobiFlight WASM Modules is installed and is up to date.
MobiFlight sees all 3 peripherals and the Loupedeck Live.
MSFS Developer Mode sees 2 registered Mobi Packages
      mobi-event-module (which is mounted) 
      and a 2nd one, mobiflight (which is unmounted)
     (I just installed MobiFlight becuase it is required.  I don't know anything about it.)
     (I do not know if both packages are needed, nor do I know how to mount a package,)
I tested with both a stock Cessna 172, and the A2A Comanche 250).  No values appear on the Loupedeck Live.  and COM & NAV dials do not change the frequencies
I know the Loupedeck Live is connected, because my custom MSFS views (key inputs) work just fine.
As I type this, I see Loupdeck has a new release.  I don't want to upgrade until advised to do so..
Any help or suggestions will be appreciated.  Thank you.  Dave In Ocean, NJ

**Comment by calibx** [comment]
@NJTravelerDH : there was a mistake in the 3.0 readme, Mobiflight is not required, it was planned but not in the 3.0.x.
I hope to release soon the 4.0 for compatibility with LoupeDeck 6.x

For your problem, can you provide the logs ?

**Comment by NJTravelerDH** [comment]
Thank your for replying,
I am not familiar with the file structure for Loupedeck or your plugin.  I have searched my computer for something related to Calibx & plugin.  I found nothing that seems to indicate a log file.  There were lots of Plugin folders, but when I opened each one, there was nothing that indicated it might be connected to Loupedeck or Calibx.  
I did find a Calibx plugin Install log from October 9th, 2022.  I don't think that is what you need.
Is there a Plugin log for your plugin?  If so, how do I start it, or find it?
You say "MobiFlight is not required".  Should I delete it?
Loupedeck:
I think I am using 5.9.1.19364, but I find there is another version, 6.0.0.20753, in my Control Panel > Apps > Installed Apps.  A few weeks ago, I upgraded, 6.0.1. and then went back to 5.9.1 because of the notes in Github.  
How do I know which version is being run when I simply tell my computer to run Loupedeck?  
Loupedeck has a new version 6.0.1.20873.  Should I upgrade Loupedeck?
I did find that Loupedeck has a "Start Log" command.  Do you want the Loupedeck log.  If so, I'll start it and send it to you.
I am trying to solve my problem, but I don't have sufficient knowledge to understand everything needed to resolve it.  I hope my questions are helpful,
Sincerely, Dave Hollinger NJTravelerDH / NJPilot in Ocean NJ

**Reply by NJTravelerDH** [reply]
I just discovered that the Loupedeck version is in About on the task bar.  Loupedeck is 6.0.0.20753.  It seems my downgrade to 5.9.1 did not work.  I will downgrade tonight when I have time, and will report back.  That may be the source of the problem. Dave Sunday Oct. 27, 3:40 pm EST.

**Comment by NJTravelerDH** [comment]
I downgraded and everything seems to work.  A few issues, RPM reads low compared to panel RPM, for example.  But I need to learn and work with it more.
I learned a lot - sorry to bother you Calibx.  Apparently, the downgrade did not work, and I didn't that, nor did I know how to check the version.
Tonight, I uninstalled both 6.0.0 and 5.9.1.  The I reinstalled 5.9.1.  I got the above results
I'll leave MobiFlight installed, since it doesn't seem to impact operation and will be needed in the future.
I'd still like to know which logs are needed, and how to find them.  If you have time let me know (more learning)
THANKS AGAIN FOR YOUR ATTENTION AND HELP !!! Dave.

**Comment by calibx** [comment]
Good news !

**Reply by lxndrb22** [reply]
We remain hopeful...

**Comment by calibx** [comment]
Just a little update : I'm still trying to load the dll but without success for now. I'll try to contact loupedeck for some help.
Bad news : they removed the 5.9 from their download section :(

**Reply by NJTravelerDH** [reply]
Thanks Calibx for your efforts.  I have a copy of 5.9 if you would like to get it.  You have my email address.  Dave in NJ.

**Comment by Aaron-C-Reskew** [comment]
The 5.9.1.download is at the top section of: https://loupedeck.com/uk/downloads/ below the 6.0.1 download

**Comment by calibx** [comment]
The 4.0.0 is compatible with Loupedeck 6 and will go to the market !
